catbuffer/parser/Jenkinsfile

```diff
@@ -11,5 +11,5 @@ defaultCiPipeline {
 	codeCoverageTool = 'coverage'
 	minimumCodeCoverage = 95
 
-	discordWebHookUrlId = 'SDK_DISCORD_WEB_HOOK_URL_ID'
+	discordWebHookUrlId = 'CATBUFFER_DISCORD_WEB_HOOK_URL_ID'
```

Comment:
is there reason we want separate channel vs sdk?

Reply:
@0x6861746366574 your thoughts?